web/app/themes/xrnl/template-parts/citizen-assembly-header.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * The template part for displaying header in citizen assembly pages
+ */
+
+$current_lang = apply_filters( 'wpml_current_language', NULL );
+
+if($current_lang == 'en') {
+    $backButtonUrl = '/en/citizen-assembly';
+    $backButtonText = 'Back';
+} else {
+    $backButtonUrl = '/burgerberaad';
+    $backButtonText = 'Terug';
+}
+
+?>
+
+<div class="join bg-yellow">
+  <div class="text-left pt-3">
+    <a id="back-button" href="<?php echo $backButtonUrl?>">
+        <h1 id="back-text"class="display-5 text-uppercase font-xr">
+            <i class="fas fa-chevron-left fa-xs px-3"></i><?php echo $backButtonText ?>
+        </h1>
+    </a>
+  </div>
+  <div class="text-center py-5 mx-auto" style="max-width:80%">
```

Comment:
In the header of the home page there is equal padding around the text, but in `citizen-assembly-header.php` the padding is not equal because the `<div>` used for the link adds additional vertical space. You can easily solve this with this change:

```suggestion
  <div class="text-center pb-5 pt-1 mx-auto" style="max-width:80%">
```